sdk/tests/deqp/modules/shared/glsShaderLibrary.js

```diff
@@ -145,9 +145,7 @@ var gluShaderUtil = framework.opengl.gluShaderUtil;
         if (trimFront)
             str = str.replace(/^\s*\n/, '');
         return str
-            .replace('\\n', '\n')
-            .replace('\\t', '\t')
-            .replace(/\\/g, '');
+            .replace(/\\\\/g, '\\');
```

Comment:
It's difficult to do replacement for '\n', '\t', '\\' with repalce() at the same time. Align the code with C++.